module-domain/src/main/kotlin/site/yourevents/invitation/port/in/InvitationUseCase.kt

```diff
@@ -11,4 +11,6 @@ interface InvitationUseCase {
     fun findById(id: UUID): Invitation?
 
     fun getQrCodeUrl(id: UUID): String
+
+    fun deleteInvitation(invitationId: UUID, deleted: Boolean)
```

Comment:
위에서 언급한 것과 비슷한 맥락입니다. 삭제 API에서 초대장이 실제로 삭제되는 것이 아닌, soft-delete 형태로 처리하기에, 메서드명을 보다 더욱 직관적으로 작성하면 덜 헷갈리지 않을까요? 어떻게 생각하시나요ㅎㅎ 😄 

그리고 매개변수로 상태값을 전달하기보다는, 도메인 내부의 비즈니스 로직에서 처리하는 게 좋아보입니다!
```suggestion
    fun markInvitationAsDeleted(invitationId: UUID)
```

Reply:
저도 정완님 의견에 동의합니다. 메서드명을 수정하는 쪽이 더 좋아보입니다. deleted 값의 경우도 그냥 없어도 비즈니스 로직에서 이미 처리가 진행되기에 삭제하도록 하겠습니다.